suite-common/suite-utils/src/pollingController.ts

```diff
@@ -0,0 +1,39 @@
+type PollingCallback = () => Promise<void> | void;
+
+export const createPollingController = () => {
+    let intervalId: ReturnType<typeof setInterval> | null = null;
+    let running = false;
+
+    const start = (cb: PollingCallback, intervalMs: number) => {
+        if (intervalId) return;
+        intervalId = setInterval(async () => {
+            if (running) return;
+            running = true;
+            try {
+                await cb();
+            } finally {
```

Comment:
Currently it silences errors. For debugging purposes, how about this? 
```suggestion
            } catch(err) { console.error(err); } finally {
```